app/components/agenda/agenda-header/agenda-version-actions.js

```diff
@@ -365,6 +365,7 @@ export default class AgendaAgendaHeaderAgendaVersionActions extends Component {
     }
     try {
       await approveAgendaAndCloseMeeting(this.args.currentAgenda);
+      await this.stampDocuments();
```

Comment:
I'm a bit worried that we don't inform the user of what we're doing here. I'll modify the translation key of the modal that says the agenda is being approved